crates/types/src/signature_key.rs

```diff
@@ -0,0 +1,132 @@
+//! Types and structs for the hotshot signature keys
+
+use crate::{
+    qc::{BitVectorQC, QCParams},
```

Comment:
Originally we imported this from `hotshot_qc::bit_vector_old`, does this refactor also move things from `hotshot_qc/src/bit_vector.rs` to `types/src/qc.rs`? If so, this PR can also close https://github.com/EspressoSystems/HotShot/issues/2070.

Reply:
No. I moved `bit_vector_old.rs`. 
I think the current stake table scheme is richer than hotshot needs. All consensus need here is a stake table snapshot, see #2375 . So `bit_vector_old.rs` might be the best choice here.

Reply:
`hotshot-qc` crate is no longer in the workspace. So the hotshot need nothing from `hotshot-qc` directory.

I'm just keeping those, because we may need the SNARKed QC implementation in the future.